app/views/ansible_roles/index.html.erb

```diff
@@ -0,0 +1,30 @@
+<% title _("Ansible Roles") %>
+
+<% title_actions ansible_proxy_import(hash_for_import_ansible_roles_path) %>
+
+<table class="<%= table_css_classes 'table-fixed' %>">
+  <thead>
+    <tr>
+      <th class="col-md-3"><%= sort :name, :as => s_("Role|Name") %></th>
+      <th class="col-md-3"><%= _("Imported at") %></th>
+      <th class="col-md-1"><%= _("Actions") %></th>
+    </tr>
+  </thead>
+  <tbody>
+    <% for role in @roles %>
```

Comment:
`.each` please, to properly scope the content of the loop (+ performance, let's save the planet :smile:)